src/index.js

```diff
@@ -29,7 +29,7 @@ export default withStyles(styles, { withTheme: true })((props) => (
 ));
 
 export {
-  MaterialTable as MTable,
+  // MaterialTable as MTable,
```

Comment:
Why is this export removed?

Reply:
I can't remember if I did this or not, but from what I can tell MaterialTable isn't exported from './components' - even outside of that, wouldn't it be redundant since MaterialTable is the default export? Why have it as default and included in named exports?

It is not currently being exported as of v2.3.39 so it doesn't look like it's causing issues.

Reply:
Ok sounds good.